### PR TITLE
Resolutions support in gaStylesFromLiterals

### PR DESCRIPTION
**Pull request by loicgasser**
https://github.com/geoadmin/mf-geoadmin3/issues/3378

ping @AFoletti You can use this branch for your test with the resolution dependent styles for the bike geojson (you need to change the BOD), it should be ready.
ping @oterral for the review.

### CODE REVIEW

File: src/components/StylesFromLiteralsService.js [hunk 1]
```diff
@@ -104,16 +104,40 @@ goog.provide('ga_stylesfromliterals_service');
...
+              minResolution: minResolution,
+              maxResolution: maxResolution
+            };
+            if (!this.styles[value.geomType][value.value]) {
+              this.styles[value.geomType][value.value] = [olStyle];
+            } else {
+              this.styles[value.geomType][value.value].push(olStyle);
+            }
           }
         } else if (this.type === 'range') {
```
Comment (oterral) [inline src/components/StylesFromLiteralsService.js]: The content of the 2 `else if` seems very close. any chance to avoid this duplication of code ?

**Comment by oterral** [comment]
A minor comment to avoid duplication of code otherwise lgtm.
I've not tested.

File: src/components/StylesFromLiteralsService.js (hunk 1 above)
Comment (loicgasser) [inline src/components/StylesFromLiteralsService.js]: Sure, I ll think of something to avoid it.

**Comment by loicgasser** [comment]
Thanks you for the review !

**Comment by gjn** [comment]
Merging this for @AFoletti and @simonevonheute .

**Pull request merged**